frontend/simple/views/components/Voting/errors.js

```diff
@@ -0,0 +1,3 @@
+export const CastError = 'cast'
+export const TresholdError = 'treshold'
+export const CloseError = 'close'
```

Comment:
IMO constants should be `MACRO_CASE`, at least it's a practice / pattern I use to see around